src/plugins/enzymes/src/FindEnzymesDialog.cpp

```diff
@@ -1021,6 +1046,38 @@ QString EnzymeTreeItem::generateEnzymeTooltip() const {
     return TOOLTIP_TAG.arg(forwardTooltip).arg(reverseTooltip);
 }
 
+QString EnzymeTreeItem::getTypeInfo() const {
+    auto type = text(Column::Type);
+    QString result;
+    static constexpr const char* UNEXPECTED_TYPE_ERROR = QT_TR_NOOP("Unexpected enzyme type: %1");
+    if (type == "M") {
+        result = tr("An orphan methylase,<br>not associated with a restriction enzyme or specificity subunit");
+
+    } else if (type.size() == 2) {
+        if (type == "IE") {
+            result = tr("An intron-encoded (homing) endonuclease");
+        } else if (type.startsWith("R")) {
+            result = tr("The restriction enzyme of the %1 type").arg(type.back());
+        } else if (type.startsWith("M")) {
+            result = tr("The methylase of the %1 type").arg(type.back());
+        } else {
+            coreLog.error(tr(UNEXPECTED_TYPE_ERROR).arg(type));
```

Comment:
This is not an error at all. 
I would even avoid logging it into trace because it may create a lot of spam in the log. 
User already sees the type/absence of the type in the text view.

Reply:
OK